### PR TITLE
Clean up TFOptimizer.get_updates()

### PR DESCRIPTION
**Pull request by autoih**

### CODE REVIEW

File: tensorflow/python/keras/optimizers.py [hunk 1]
```diff
@@ -728,29 +728,26 @@ def get_grads(self, loss, params):
...
-      self.updates = []
-
       if not params:
         # After the model vars have been created, the second call to get_updates
         # is called with params as an empty list. This ensures that we call
         # compute_gradients with params=None.
-        grads = self.optimizer.compute_gradients(loss)
-      else:
-        grads = self.optimizer.compute_gradients(loss, params)
+        params = None
```
Comment (tanzhenyu) [inline tensorflow/python/keras/optimizers.py]: ?

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by tanzhenyu** [comment]
This is not waiting for review, right?

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**